presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -164,8 +164,17 @@ class SystemConfig : public ConfigBase {
       "task.max-drivers-per-task"};
   static constexpr std::string_view kConcurrentLifespansPerTask{
       "task.concurrent-lifespans-per-task"};
-  static constexpr std::string_view kHttpExecThreads{"http_exec_threads"};
-  static constexpr std::string_view kNumHttpCpuThreads{"num-http-cpu-threads"};
```

Comment:
This means the new deployment scripts need to pass new name. And new deployment script can not be used to update existing binary anymore because existing binary does not recognize new name. This might require adding versioning in deployment scripts.

Reply:
@amitkdutta 
We discussed this. We don't need the versioning here.
Test and dev environment would use the default values for a very short time.
Prod won't be pushed.
We'll update the scripts as soon as this one lands.